### ISSUE TITLE
FileStream doesn't work well without piping

### ISSUE THREAD

**Issue by D8207** [maxogden/filereader-stream#11]
My real world use case is passing it to https://github.com/kunklejr/node-pcap-parser/blob/master/lib/pcap-parser.js

The first problem I got is `Uncaught TypeError: Cannot read property 'readAsArrayBuffer' of undefined`. The parser calls .resume() on FileStream directly without .pipe(), so .reader doesn't get initialized, causing this error.

The second problem is `Uncaught TypeError: Cannot read property 'write' of undefined`. Like the problem above, .dest only gets initialized in .pipe(), so calling .pause() / .resume() directly leaves it undefined.

**Comment by D8207** [comment]
My workaround is:
```
var stream = createReadStream( e.data );

// https://github.com/maxogden/filereader-stream/issues/11
stream.reader = stream._FileReader();
stream.dest = {
        write: function( data ) {
                stream.emit( 'data', data );
        }
};

var parser = pcapp.parse( stream );
```

**Comment by max-mapper** [comment]
Fixed as of 1.0.0
